app/src/main/cpp/skyline/services/timesrv/time_shared_memory.h

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <common.h>
+#include <common/uuid.h>
+#include <kernel/types/KEvent.h>
+#include <kernel/types/KSharedMemory.h>
+#include <services/timesrv/common.h>
+
+namespace skyline::service::timesrv::core {
+    struct TimeSharedMemoryLayout;
```

Comment:
As determined by our (somewhat informal) convention, if there's any links to a structure from a header then that structure should be placed in the header as well

Reply:
Hm? Disagree, placing it in the c++ file makes much more sense here

Reply:
Especially considering the locality to accessor functions etc, if you take this as a rule you would need GlobalServiceState to be in header (which wouldn't be very good at all)

Reply:
"Smh you're wrong but I concede"